identity-wallet/src/state/credentials/reducers/send_credential_request.rs

```diff
@@ -246,5 +263,11 @@ pub async fn send_credential_request(mut state: AppState, action: Action) -> Res
         });
     }
 
-    Ok(state)
+    Ok(AppState {
+        core_utils: CoreUtils {
+            managers: state.core_utils.managers,
+            ..Default::default()
```

Comment:
default or needs to be filled with ..state.core_utils

Reply:
thanks, fixed :+1: